admin/form-builder/assets/js/components/form-fields-v4-1/index.js

```
@@ -0,0 +1,220 @@
+                    if (result.isConfirmed) {
+                        window.open(wpuf_form_builder.pro_link, '_blank');
+                    }
+
+                }, function() {});
+            }
+        },
+
+        alert_invalidate_msg: function (field) {
+            var validator = this.field_settings[field].validator;
+
+            if (validator && validator.msg) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)